examples/kitchen-sink/__test__/build.test.js

```diff
@@ -20,7 +24,6 @@ const expected = [
   'regenerator-runtime.bundle.js',
   'scripts.bundle.js',
   'shared.stable.bundle.js',
-  'shared.volatile.bundle.js',
   'styles.css',
   'webpack-runtime.bundle.js'
 ]
```

Comment:
How did changes to the Handlebars helpers result in a change to the expected kitchen sink build JS manifest files?